src/mme-app/conf/mme.json

```diff
@@ -19,6 +19,10 @@
 			"dig3": -1
 		}
 	},
+	"enb": {
```

Comment:
wondering what MME does with this config ?

Reply:
WE have some helmchart dependency and we are kind of getting rid of this configuration from the code. Or at least we need to add corresponding config in the mme helm chart.